docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+- **You can make decisions even with little data.** In the context of small samples, it can often be very difficult to achieve statistical significance using frequentist methods. Since Bayesian methods produce a whole distribution for the treatment effect, you can still make a decision as long as you are willing to adjust your level of rigor to match the level of signal you can get from your data: if zero is within the 95% credible interval, then you can't be 95% sure that the treatment had a positive effect, but if you can say that there's a 70% chance that the lift is bigger than the MDE of 1%, perhaps that's good enough to make a ship decision given the limitations of the experiment.
+- **You can make a decision at any time.** Bayesian analysis is [not immune to peeking](http://varianceexplained.org/r/bayesian-ab-testing/), but it does avoid the issue simply by making no promises about the false positive rate. The idea of there being a "true effect" or "no effect" doesn't really make sense in a Bayesian paradigm: the lift is not simply zero vs. non-zero, but rather has a continuous *distribution* of values; since there is no "positive result" or "negative result", there can't be a *false* positive or *false* negative.
+
+#### Cons of Bayesian analysis
+
+- **Stakeholders might not be used to the Bayesian way of thinking.** Although some Bayesian concepts are more intuitive than their frequentist counterparts, frequentism is still the more common paradigm people learn for doing statistical inference, so switching from "This metric had a statistically significant increase" to "The treatment is 95% likely to have increased this metric" might require educating stakeholders on how to use Bayesian results.
+- **You have to decide *how* to make decisions.** The output of Bayesian inference is a statistical distribution, rather than a "yes"/"no" answer to the question "Was there a statistically significant effect?" This means that decision-makers need to align on how to translate that distribution into a decision on whether to ship a particular treatment. For example, do you care more about risk, or probability the treatment beats control? What probability that the lift exceeds the MDE is considered "good enough to ship"?
```

Comment:
Since we show a credible interval, which is pretty similar to a confidence interval, these can be used in the same way no?